archunit-junit/junit5/engine/src/main/resources/META-INF/services/org.apache.maven.surefire.junitplatform.TestSelectorFactory

```diff
@@ -0,0 +1 @@
+com.tngtech.archunit.junit.surefire.FieldSelectorFactory
```

Comment:
The idea is that both Gradle Test Plugin and Surefire now define a `TestSelectorFactory` as an SPI, as well as a `MethodSelectorFactory` as the default implementation. Additional implementations (like this one defined here) can then be picked up from the classpath, but the default `MethodSelectorFactory` cannot be resigned from in any way